### ISSUE TITLE
[0.6.0] securedrop-admin setup fails to install apt dependencies on Tails 3.6~rc1

### ISSUE THREAD

**Issue by redshiftzero** [freedomofpress/securedrop#3108]
## Description

A short summary of the issue.

## Steps to Reproduce

From a fresh Tails 3.6~rc1 VM:

```
amnesia@amnesia:~/Persistent/securedrop$ ./securedrop-admin setup
```

## Expected Behavior

All is merry and well and the virtualenv is set up

## Actual Behavior

```
INFO: Installing SecureDrop Admin dependencies
...[snip]...
Building dependency tree...
Reading state information...
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 libpython2.7-dev : Depends: libexpat1-dev but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
ERROR: Failed to install apt dependencies. Check network connection and try again.
```

## Comments

```
dpkg --get-selections | grep hold
```

returns nothing

**Comment by conorsch** [comment]
Curious whether the `-v` option would provide sufficient verbosity to debug here—otherwise we'll have to run the `apt-get install` command manually with the list of packages. I suspect apt pinning preferences are the culprit here, hopefully something that will be shaken out prior to stable, but definitely something we want to understand fully as soon as possible.

**Comment by singuliere** [comment]
libc6 in tails 3.6 is 2.24.11+deb9u3 (from sid) and apt-get install libc6-dev (from stretch) will fail because it depends on 2.24.11+deb9u1. apt-get install libpython2.7-dev will also fail for the same reason.

https://labs.riseup.net/code/issues/15383

**Comment by kushaldas** [comment]
It seems to be a packaging issue in tails/debian side.

```
$ sudo apt install libexpat1-dev
[sudo] password for amnesia: 
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 libexpat1-dev : Depends: libc6-dev but it is not going to be installed or
                          libc-dev
E: Unable to correct problems, you have held broken packages.
```

```
$ sudo apt install libc6-dev
[sudo] password for amnesia: 
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 libc6-dev : Depends: libc6 (= 2.24-11+deb9u1) but 2.24-11+deb9u3 is to be installed
E: Unable to correct problems, you have held broken packages.
```

**Comment by kushaldas** [comment]
Temporary solution: `As a temporary workaround you can add stretch-proposed-updates to your APT sources.`

Will be fixed in the next build of Tails. Details are in https://labs.riseup.net/code/issues/15383.

**Comment by kushaldas** [comment]
```
deb tor+http://vwakviie2ienjx6t.onion/debian/ stretch-proposed-updates main contrib
```

I have added the above line in my sources.list, setup worked well.

**Comment by redshiftzero** [comment]
Great, thanks @singuliere and @kushaldas - I'll leave this open for any other QA-ers and we can close when the next Tails `rc` is out

**Comment by redshiftzero** [comment]
This was fixed in the final version of Tails 3.6, closing
